addons/im_livechat/static/tests/embed/history_command.test.js

```diff
@@ -26,12 +24,11 @@ test("Handle livechat history command", async () => {
         return true;
     });
     await start({ authenticateAs: false });
-    await mountWithCleanup(LivechatButton);
     await click(".o-livechat-LivechatButton");
     await contains(".o-mail-Composer-input").edit("Hello World!", { confirm: false });
     await press("Enter");
     await waitFor(".o-mail-Message:contains(Hello World!)");
-    const thread = getService("im_livechat.livechat").thread;
+    const thread = Object.values(getService("mail.store").Thread.records).at(-1);
     const guestId = pyEnv.cookie.get("dgid");
     const [guest] = pyEnv["mail.guest"].read(guestId);
     pyEnv["bus.bus"]._sendone(guest, "im_livechat.history_command", {
```

Comment:
For now, we should implement the mock server, rather than having to do that manually.

Reply:
That's not linked to the mock server? We would need the operator to actually trigger the `/history` command which is not possible with our test configuration (two different users on two different pages). This test just ensures the history to a given channel is returned when requested. But to do so, we need to specify the channel id. That's not pretty but I don't have another solution to grab it.